samsung_mdc/utils.py

```diff
@@ -52,3 +52,20 @@ def repr_hex(value):
 
 def parse_hex(value):
     return value and bytes(int(x, 16) for x in value.split(':')) or b''
+
+
+def parse_videowall(value):
+    """
+    Splits 8 byte 'coordinates' to 2x4 byte x/y tuple
+    The expected representation is Y, X
+    """
+    return divmod(value, 1<<4)[::-1]
```

Comment:
This will only work if the `value` I get here is hex in the form `0xff`. Is this the case?